app/uk/gov/hmrc/exports/controllers/request/ExportsDeclarationRequest.scala

```diff
@@ -191,13 +80,8 @@ object ExportsDeclarationRequest {
       (__ \ "totalNumberOfItems").readNullable[TotalNumberOfItems] and
       (__ \ "previousDocuments").readNullable[PreviousDocuments] and
       (__ \ "natureOfTransaction").readNullable[NatureOfTransaction]
-  ).apply(ExportsDeclarationRequest.version2 _)
-
-  val bothReads: Reads[ExportsDeclarationRequest] = (__ \ "transport").readNullable[Transport].flatMap[ExportsDeclarationRequest] {
-    case Some(_) => readsVersion2
-    case None    => readsVersion1
-  }
+  ).apply(ExportsDeclarationRequest.apply _)
 
   implicit val format
-    : OFormat[ExportsDeclarationRequest] = OFormat(bothReads, Json.writes[ExportsDeclarationRequest]) // writes are used only for logging
+    : OFormat[ExportsDeclarationRequest] = OFormat(readsVersion2, Json.writes[ExportsDeclarationRequest]) // writes are used only for logging
```

Comment:
I thought we were removing the explicit reads/writes.
@PatrykRudnicki  please confirm.

Reply:
Output of the meeting was to come back to implementation before, without manually written reads and writes